lib/generators/provider/templates/.travis.yml

```diff
@@ -9,6 +9,7 @@ env:
   - RUBY_GC_HEAP_GROWTH_MAX_SLOTS=300000
   - RUBY_GC_HEAP_INIT_SLOTS=600000
   - RUBY_GC_HEAP_GROWTH_FACTOR=1.25
+bundler_args: --no-deployment
```

Comment:
reading back to https://github.com/ManageIQ/manageiq-ui-classic/pull/1606#issuecomment-311386307 it seems, this only affects the ui and miq-content repo. 
As the provider repos use `install: bin/setup` in `.travis.yml` and the others dont.

I think we should streamline this and all plugins should use `bin/setup` as the install